configstore/tests/test_docker_secret.py

```diff
@@ -11,7 +10,9 @@
 
 class TestDockerSecretBackend(TestCase):
 
-    def test_get_secret(self):
+    @mock.patch('configstore.backends.docker_secret.os.path.exists',
```

Comment:
`configstore.backends.etc.os` == `os`, so you could simply patch `os.path.exists`.